### PR TITLE
Separate gAMA/cHRM chunk and contents

### PR DESCRIPTION
**Pull request by HeroicKatora**
Introduces two new fields to `Info`: `gama_chunk` and `chrm_chunk` which hold the _original_ contents of those chunks regardless of any overwriting/ignore by sRGB. They are also used to track if such a chunk was already parsed instead of reusing the shared fields. Since the struct is `#[non_exhaustive]` this is a backwards compatible change.

Closes: #340 (still needs a regression test)

### CODE REVIEW

**Comment by HeroicKatora** [comment]
I'll already preparing a changelog and release PR directly afterwards, let's discuss the changelog item there.

File: tests/bugfixes/License.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[[files]]
+file = "tests/bugfixes/gama-srgb-order-issue#304.png"
+SPDX-FileCopyrightText = "2013 zaimoku_woodpile <https://www.flickr.com/photos/11250735@N07/>"
+SPDX-License-Identifier = "CC-BY-2.0"
+Comment: """
+	Taken from: <https://www.flickr.com/photos/11250735@N07/8561945042>
+	with Modifications: Cropped, resized the image
+	Taken from: <https://raw.githubusercontent.com/slint-ui/slint/v0.1.6/examples/imagefilter/cat_preview_round.png>
+"""
```
Comment (fintelia) [inline tests/bugfixes/License.toml:14]: Is the syntax here correct? GitHub is highlighting it in red...

Comment (HeroicKatora) [reply tests/bugfixes/License.toml:14]: I would [assume so](https://github.com/Randy8080/reference/blob/main/toml.md#multiline-string) and my editor is also highlighting this properly.

Comment (HeroicKatora) [reply tests/bugfixes/License.toml:14]: Oops, that's why you're referring to. Yeah..

**Pull request merged**